Golem.Tests/JobTests.cs

```diff
@@ -26,53 +27,34 @@ public async Task CompleteScenario()
 
             string golemPath = await PackageBuilder.BuildTestDirectory();
             _logger.LogInformation($"Path: {golemPath}");
-            var golem = await TestUtils.Golem(golemPath, _loggerFactory, null, RelayType.Local);
+            await using var golem = (Golem)await TestUtils.Golem(golemPath, _loggerFactory, null, RelayType.Local);
 
-            Channel<GolemStatus> golemStatusChannel = PropertyChangeChannel(golem, nameof(IGolem.Status),
-                (GolemStatus v) => _logger.LogInformation($"Golem status update: {v}"));
-
-            var jobStatusChannel = PropertyChangeChannel<IJob, JobStatus>(null, "");
-            var jobPaymentStatusChannel = PropertyChangeChannel<IJob, GolemLib.Types.PaymentStatus?>(null, "");
-            var jobGolemUsageChannel = PropertyChangeChannel<IJob, GolemUsage>(null, "");
-            var jobPaymentConfirmationChannel = PropertyChangeChannel<IJob, List<Payment>>(null, "");
-
-            Channel<Job?> jobChannel = PropertyChangeChannel(golem, nameof(IGolem.CurrentJob), (Job? currentJob) =>
-            {
-                _logger.LogInformation($"Current Job update: {currentJob}");
-
-                jobStatusChannel = PropertyChangeChannel(currentJob, nameof(currentJob.Status),
-                    (JobStatus v) => _logger.LogInformation($"Current job Status update: {v}"));
-                jobPaymentStatusChannel = PropertyChangeChannel(currentJob, nameof(currentJob.PaymentStatus),
-                    (GolemLib.Types.PaymentStatus? v) => _logger.LogInformation($"Current job Payment Status update: {v}"));
-                jobGolemUsageChannel = PropertyChangeChannel(currentJob, nameof(currentJob.CurrentUsage),
-                    (GolemUsage? v) => _logger.LogInformation($"Current job Usage update: {v}"));
-                jobPaymentConfirmationChannel = PropertyChangeChannel(currentJob, nameof(currentJob.PaymentConfirmation),
-                    (List<Payment>? v) => _logger.LogInformation($"Current job Payment Confirmation update: {v}"));
-
-            });
```

Comment:
This handler updating various Job property channels was a cause of race condition.
Test case could later grab reference of e.g. `jobStatusChannel` property before it got updated by handler.